map/src/context/TracksManager.js

```diff
@@ -1212,6 +1213,10 @@ export const getGpxTime = (f, reverse = false) => {
     return 0;
 };
 
+export function prepareDesc(trackDesc) {
+    return anchorme(trackDesc.replace(/\n/g, '<br />'));
+}
```

Comment:
Please add check trackDesc.

Selenium tests failed (no .replace in trackDesc).

Also, Jest-test should have more input data such as: [null, '', undefined, ...]